### DISCUSSION TITLE
How often does the elastalert2 poll the elastic?

### DISCUSSION THREAD

**Discussion by alexku7** [jertel/elastalert2#571] [Q&A] [answered]
Can't fine how often the elastalert2 polls the elastic

Is this configurable ?

Plse help :)

**Comment by jertel** [comment, marked as answer]
Default is every 5 minutes. Overridable via `run_every` as mentioned in the [Configuration](https://elastalert2.readthedocs.io/en/latest/elastalert.html?highlight=run_every#configuration) chapter of the docs (very first section) and again in the second section: [Running ElastAlert 2](https://elastalert2.readthedocs.io/en/latest/running_elastalert.html?highlight=run_every#downloading-and-configuring).

Please review #11.

**Reply by alexku7** [reply]
Thanks, Found it :)
